docker/lib/dependabot/docker/update_checker.rb

```diff
@@ -294,6 +294,10 @@ def fetch_digest_of(tag)
       rescue RestClient::ServerBrokeConnection,
              RestClient::TooManyRequests
         raise PrivateSourceBadResponse, registry_hostname
+      rescue JSON::ParserError
```

Comment:
do we know why this happens? For example, why is the tag empty?

Reply:
I don't know at this point but one of the reasons I added the log below is so that I can get the docker repo name and the tag as that would be handy in any further investigations.